backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1002,9 +999,8 @@ void convertToServiceRequest_Strings_null() {
   @Test
   void convertToServiceRequest_TestOrder_matchesJson() throws IOException {
     var internalId = "3c9c7370-e2e3-49ad-bb7a-f6005f41cf29";
-    var testOrder = TestDataBuilder.createEmptyTestOrder();
+    var testOrder = TestDataBuilder.createTestOrderWithMultiplexDevice();
```

Comment:
For anyone else wondering why we don't need to update the `serviceRequest.json` file is because this method creates a device with the same test ordered code as the value it previously used!